### PR TITLE
Public preview of keep with latest

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?
Public preview launch of the **Keep with latest version** setting in dbt Cloud.

Update pills from "beta" to "public preview" and applicable callouts.

Updated existing pages:
- Upgrade to v1.8
- Update dbt version in dbt Cloud
- Checking version compatibility page
- require-dbt-version

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Needs review from product team

### CODE REVIEW

File: website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md [hunk 1]
```diff
@@ -11,19 +11,18 @@ Navigate to the settings page of an environment, then click **Edit**. Click the
...
 
 By choosing to **Keep on latest version**, you always get the latest fixes and early access to new functionality for your dbt project. dbt Labs will handle upgrades for you, as part of testing and redeploying the dbt Cloud SaaS application.
 
 You can upgrade to **Keep on latest version** no matter which version of dbt you currently have selected. As a best practice, dbt Labs recommends that you test the upgrade in development first; use the [Override dbt version](#override-dbt-version) setting to test _your_ project on the latest dbt version before upgrading your deployment environments and the default development environment for all your colleagues.
 
-:::tip Interested? 
+:::note Availability  
 
-This feature is currently available in beta for all dbt Labs-maintained adapters and will be widely available in March.
+Public preview is currently available for Databricks and Starburst/Trino connections. Additional support will be coming in late March.
```
Comment (jtcohen6) [inline website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md]: Let's just call out Fabric specifically as coming soon. Something like:
```suggestion
[Microsoft Fabric support](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-microsoft-fabric) will be coming in late March. All other connections are supported.
```

**Review by jtcohen6 — COMMENTED** [review]
Looks good!
- One comment about note re: adapter/connection support
- As discussed offline, update screenshot to remove `(beta)` callout = what this will look like when dbt Cloud redeploys :)

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md [hunk 2]
```diff
@@ -17,11 +17,13 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
...
 
-## Keep on the latest version in dbt Cloud
+## Keep on the latest version <Lifecycle status='public preview' />
 
-With dbt Cloud, you can get early access to many new features and functionality before it's in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. For more details, refer to [Upgrade Core version in Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud).
+With dbt Cloud, you can get early access to many new features and functionality before it's in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. Refer to the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud) setting for more details.
 
-The **Keep on latest version** setting is currently available in beta for most dbt Cloud customers, with full availability by the end of March 2024.
+:::note Availability  
+Public preview is currently available in dbt Cloud for Databricks and Starburst/Trino connections. Additional support will be coming in late March.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]:
```suggestion
[Microsoft Fabric support](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-microsoft-fabric) will be coming in late March. All other connections are supported.
```

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md [hunk 3]
```diff
@@ -17,11 +17,13 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 dbt Labs plans to release dbt Core v1.8.0-b1 and dbt Labs-maintained adapters on February 28th, 2024.
 
-## Keep on the latest version in dbt Cloud
+## Keep on the latest version <Lifecycle status='public preview' />
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]:
```suggestion
## Keep on latest version <Lifecycle status='public preview' />
```

**Review by jtcohen6 — APPROVED** [review]

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md [hunk 4]
```diff
@@ -17,11 +17,13 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 dbt Labs plans to release dbt Core v1.8.0-b1 and dbt Labs-maintained adapters on February 28th, 2024.
 
-## Keep on the latest version in dbt Cloud
+## Keep on latest version <Lifecycle status='public preview' />
 
-With dbt Cloud, you can get early access to many new features and functionality before it's in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. For more details, refer to [Upgrade Core version in Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud).
+With dbt Cloud, you can get early access to many new features and functionality before it's in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. Refer to the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud) setting for more details.
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]:
```suggestion
With dbt Cloud, you can get early access to many new features and functionality before they're in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. Refer to the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud) setting for more details.
```

**Review by runleonarun — APPROVED** [review]
This looks great! Just one tiny suggestion. 🚢 it!

**Pull request merged**